### DISCUSSION TITLE
New translation in German

### DISCUSSION THREAD

**Discussion by stepstra** [sircharlo/meeting-media-manager#257] [Translations]
I would like to help to translate JWMMF into a language I speak, German (X/de).

**Comment by sircharlo** [comment]
> I would like to help to translate JWMMF into a language I speak, German (X/de).

Thanks for reaching out @stepstra !
If you're comfortable forking a GitHub repo, feel free to do so and submit a PR with a `de.json` file added to the `locales` folder, with the contents copied from `en.json` and translated into your language.
If not, feel free to simply download [the english locale file](https://github.com/sircharlo/jw-meeting-media-fetcher/blob/master/locales/en.json) from [here](https://raw.githubusercontent.com/sircharlo/jw-meeting-media-fetcher/master/locales/en.json), translate the strings, and save and upload your translation in this discussion. 
Thanks for your willing spirit!

**Reply by stepstra** [reply]
Hi. I think so but haven't done it for quite a while. So I post the file here for now. Would be great if you could send me a test package or just show me how to integrate the file in my existing installation.
[de.zip](https://github.com/sircharlo/jw-meeting-media-fetcher/files/8507277/de.zip)

**Reply by sircharlo** [reply]
Thanks for the translation! I've committed it. You can test it locally before the next release by cloning the repo and running the app using `npm i; node run start`.

**Reply by stepstra** [reply]
[de.zip](https://github.com/sircharlo/jw-meeting-media-fetcher/files/8546446/de.zip)
New version with missing lines

**Reply by sircharlo** [reply]
Thank you !
